app/src/main/java/org/greenstand/android/TreeTracker/fragments/TreePreviewFragment.kt

```diff
@@ -10,13 +10,17 @@ import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.lifecycleScope
 import androidx.navigation.fragment.navArgs
-import kotlinx.android.synthetic.main.activity_main.*
-import kotlinx.android.synthetic.main.fragment_tree_preview.*
+import java.util.Date

```

Comment:
Ignore, ktlint formatting

Reply:
Hello Arun, Are the URLs linked in the about page in the app?

Reply:
> Hello Arun, Are the URLs linked in the about page in the app?

That is a issue tracked separately. Will work on that soon.